### ISSUE TITLE
[Question] How to get the status bar look without barpadding?

### ISSUE THREAD

**Issue by fawzakin** [siduck76/chadwm#22]
Hello, I've been configuring my dwm build [here](gitlab.com/fawzakin/dwm) and I want to have status bar similar to yours (with text background and padding between the text and the edge of the bar) but I don't want to have barpadding patch. 

I tried to follow what you do in this [reddit thread](https://www.reddit.com/r/suckless/comments/on80gv/dwm_statuspadding_patch_conflicts_with_status2d/) but when I got statuspadding and status2d working, status2d's text background is still drawing over the padding (to the edge of the bar). 

I tried to go with the border bar patch and patch everything [eProTaLT83](https://www.reddit.com/user/eProTaLT83) made but in the end, the systray didn't get covered with the border and one of the icon became enlarged. It seems that the patch they made is tailored for your build which has notitle patch (I need to have window title).

I need help to get this on my own build of dwm. What patch should I use and what can I do to solve the conflict. I really like the looks of your build. Thank you.

**Comment by siduck76** [comment]
@fawzakin Sorry I cant help much , I couldnt track the patches I patched :( . one thing you could do is check the commit history of this repo and help yourself.

**Comment by fawzakin** [comment]
Where did you get the border patch exactly? I bet that's how you get the bar as in the screenshot.

**Comment by siduck76** [comment]
> Where did you get the border patch exactly? I bet that's how you get the bar as in the screenshot.

bar border you mean? many bar related patches are modified on my build by  eProTaLT83

**Comment by fawzakin** [comment]
Alright, I've finally got this working. It turns out that I can still get the look of the bar as I desire. It's just that the patches make the bar look out of place at first. 

This is what the bar initially looks like (the weirdly-sized tray icon is caused by the systray patch itself it seems):  
![borked](https://gitlab.com/fawzakin/screenpaper/raw/main/Screenshots/dwm-borked.png)

And this is what it looks like after extensive ricing:
![bar](https://gitlab.com/fawzakin/screenpaper/raw/main/Screenshots/dwm-bar-rice.png)

What I did here is I get the systray and status2d patches working and apply patches from this [commit](https://github.com/siduck76/chadwm/commit/5f8df23a1f66894a3917c1ed68ffe8838e0fe74c#diff-34f80b9bef2f6b65e5b0c27fb2eee14c4705b5d89512f4f95cdfae36c6557ad4) that are relevant to bar border and underlinetags.

Here's my build in case you are interested:
https://gitlab.com/fawzakin/dwm

Issue closed.
